superagi/helper/s3_helper.py

```diff
@@ -111,4 +111,28 @@ def upload_file_content(self, content, file_path):
         try:
             self.s3.put_object(Bucket=self.bucket_name, Key=file_path, Body=content)
         except:
-            raise HTTPException(status_code=500, detail="AWS credentials not found. Check your configuration.")
\ No newline at end of file
+            raise HTTPException(status_code=500, detail="AWS credentials not found. Check your configuration.")
+        
+    @classmethod
+    def get_download_url_of_resources(cls,db_resources_arr):
+        s3 = boto3.client(
```

Comment:
remove client creation and replace with self.s3

Reply:
done